.github/workflows/analytics.yml

```diff
@@ -22,6 +23,8 @@ jobs:
       run: scripts/setup_bundler.sh
     - name: GoogleAppMeasurement
       run: scripts/third_party/travis/retry.sh pod spec lint GoogleAppMeasurement.podspec.json
+    - name: FirebaseAnalyticsSwift
+      run: scripts/third_party/travis/retry.sh pod spec lint FirebaseAnalyticsSwift.podspec
```

Comment:
This should use the script and be a regular pod lib lint since it's a source pod, not a binary pod like GoogleAppMeasurement